docs/bbs/file-type.md

```diff
@@ -0,0 +1,59 @@
+# [Kingdom Hearts II](index.md) - file types
```

Comment:
I think you forgot to change this to Birth By Sleep ;)

Reply:
Yeah, sorry. I fixed it!